make_kernel.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -e
+cd "${0%/*}"
+
+meson kernel/build kernel && ninja -C kernel/build && echo "[*] Done, kernel saved to 'kernel/build/kernel.elf'!"
```

Comment:
Only `/build` is ignored, so `/build/kernel` would be a better pick

Reply:
yeah probably
but as you rightly suggested, we should rather put a proper root meson build instead of this dull shell script
I close the PR
sorry for the noise